src/objects/Allocation__c.object

```diff
@@ -116,6 +116,20 @@
         <trackTrending>false</trackTrending>
         <type>Lookup</type>
     </fields>
+    <fields>
+        <fullName>Payment__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>Attributes allocations to a Payment. All settled Payments roll up to this Allocation&apos;s GAU.</description>
```

Comment:
@lparrott Has docs reviewed the field label, description and inlinehelp text in the other PR?

Reply:
This was reviewed before we decided to **not** include Payment rollups. Could we just remove the second sentence?

Reply:
I don't have enough business knowledge on this feature to answer that. Maybe @juliesnow or @pedrodalcol can answer that question.

Overall, we do not package any user-visible text (field labels, field inlinehelptext, field descriptions, related list labels, custom labels) that have not been reviewed and approved by someone in docs before merging. That's why I was checking on these. Technically we are able to push new values to the Field Label and Field Description, but once the new field is deployed to a customer the value in InlineHelpText in their org can not be changed by us in future package updates. That's why there's a need for review/approval before merging.